bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,237 @@
+# MagentaTV Binding
+
+This Binding allows controlling the Deutsche Telekom Magenta TV Media Receiver series MR4xx and MR2xx (Telekom NGTV / Huawei Envision platform). 
+The Binding does NOT support MR3xx/1xx (old Entertain system based on Microsoft technology)!
+
+Media Receivers are automatically discovered.
+You could send keys as you press them on the remote and the binding receives program information when the channel is switched (no
+The binding provides device discovery, sending keys for the remote and also receiving program information/events.
+
+## Supported Models
+
+* Deutsche Telekom Media Receiver MR401B - fully supported
+* Deutsche Telekom Media Receiver MR201  - fully supported
+* Deutsche Telekom Media Receiver MR400  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR200  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR3xx  - NOT supported (different platform)
+* Deutsche Telekom Media Receiver MR1xx  - NOT supported (different platform)
+
+
+### Receiver Standby Mode
+
+The Media receiver has 3 different standby modes, which could be selected in the receiver's settings.
+
+- Standby - full standby, receiver active all the time
+- Suspend/Resume - The receiver goes to sleep mode, but could be awaked by a Wake-on-LAN packet
+- Shutdown - Powering off will shutdown the receiver, can be awaked only with the power button
+
+Standby provides the best results, because the binding could wake-up the receiver (Power On/Off). Suspend/Resume would require a Wake-on-LAN, which could be done, but is currently not implemented. 
+Shutdown turns the receiver off, which requires a manual PowerOn.
+
+There is no way to detect the "display status" of the receiver. 
+The binding detects Power-Off with the MR401B/MR201 by listening to UPnP events, but can't verify the status when started.
+You need to take care on the current status if you power on/off the receiver from scenes.
+Check the current status before sending the POWER button, because POWER is a toggle, not ON or OFF (see sample rules).
+
+### Discovery
+
+The auto discovery starts when the binding is loaded.
+UPnP will be used to discover receivers on the local network.
+Based on UPnP all necessary parameters are auto-discovered.
+The receiver needs to be powered on to get discovered.
+
+### Supported Things
+
+|Thing    |Type                                                           |
+|---------|-------------------------------------------------------------- |
+|receiver |A MagentaTV Receiver, the binding supports multiple receivers. |
+
+
+
+# Device setup
+
+Once the thing will be added from the Inbox in PaperUI you'll need your T-Online credentials to query the userID.
```

Comment:
It is not clear what "userID" refers to. The thing requires an udn as a parameter, is it this what is meant?

Reply:
I replaced it with UID to avoid confusion

In fact we binding needs to login to the portal and run the OAuth process. As results of an successful login you'll receive a so called userId, which is required to do the local pairing.

You could obtain the ID with Wireshark and set it up manually in the thing config, but of course most users have the receiver connected to the Internet anyways.